core/src/ibc/channel_04/log.rs

```diff
@@ -38,7 +38,6 @@ pub fn set_packet<'a>(
     }
```

Comment:
Typo in the commit message 'remove_pcket'

Reply:
I fixed it. Thanks!